reframe_config_bot.py

```diff
@@ -0,0 +1,58 @@
+# WARNING: this file is intended as template and the __X__ template variables need to be replaced
+# before it can act as a configuration file
+# Once replaced, this is a config file for running tests after the build phase, by the bot
```

Comment:
add author + pointer to more info (https://www.eessi.io/docs/test-suite/ReFrame-configuration-file)